### DISCUSSION TITLE
shifting the main part of the page due to long titles in the TOC

### DISCUSSION THREAD

**Discussion by tadmi** [vuepress-theme-hope/vuepress-theme-hope#2968] [General]
https://user-images.githubusercontent.com/63556979/228740678-258fde48-dc0b-401b-b46a-2957946e4399.mp4

**Comment by Mister-Hope** [comment]
![image](https://user-images.githubusercontent.com/33315834/229038234-aae048ed-e095-46bf-a97c-41fd108d998d.png)

![image](https://user-images.githubusercontent.com/33315834/229038258-5177eb14-cd43-4f87-83e7-f4e6daa0a7d0.png)

![image](https://user-images.githubusercontent.com/33315834/229038323-a56aea50-3a39-42bf-bb75-9fce96a588cd.png)


Reproduction plz, if you keep posting questions in this way, I may ignore all your discussions.

**Comment by tadmi** [comment]
Sorry, it was my bad: in index.scss i am set #toc {max-width: none;}, since the TOC is heavily cropped on a wide screen.
